diagnostic.js

```diff
@@ -41,7 +44,7 @@ db.dropDatabase();
 // House Arryn is not honorable!  Delete their motto.
 
 // your code begins here
-
+db.houses.delete({"motto": 'Fire and Blood' });
```

Comment:
This is kind of a trick question. `motto` is a part of a document, so deleting a motto is actually an UPDATE action, like this:

```js
db.houses.update({ name: "House Arryn"}, { $unset: { motto: 'anything'} });
```